docs/usage/interpreter/interpreter_binding_mode.md

```diff
@@ -25,4 +25,97 @@ limitations under the License.
 
 ## Overview
 
+<center><img src="/assets/themes/zeppelin/img/docs-img/interpreter_binding_per_note_user.png" height="70%" width="70%"></center>
+<center><img src="/assets/themes/zeppelin/img/docs-img/interpreter_binding_scoped_isolated.png" height="70%" width="70%"></center>
 
+<br/>
+
+Interpreter is a JVM process that communicates to Zeppelin daemon using thrift. 
+Each Interpreter process can have Interpreter Groups, and each interpreter instance belongs to this Interpreter Group.
```

Comment:
does this mean to be one group?
`Each Interpreter process can have an Interpreter Group`
?
or
`Each Interpreter process can have multiple Interpreter Groups`
for clarity?